app/models/labware_creators/final_tube_from_plate.rb

```diff
@@ -40,23 +40,24 @@ def anchor
 
     private
 
-    # rubocop:todo Naming/MemoizedInstanceVariableName
-    def transfer_into_existing_tubes!
-      @transfer ||= transfer_template.create!(user: user_uuid, source: parent_uuid)
+    def create_transfer!
+      @create_transfer ||= transfer!(source_uuid: parent_uuid)
     end
 
-    # rubocop:enable Naming/MemoizedInstanceVariableName
-
     def pass_tubes!
-      raise StandardError, 'Tubes cannot be passed before transfer' if @transfer.nil?
+      raise StandardError, 'Tubes cannot be passed before transfer' if @create_transfer.nil?
 
       tubes_from_transfer.each do |tube_uuid|
-        api.state_change.create!(user: user_uuid, target: tube_uuid, target_state: 'passed')
+        Sequencescape::Api::V2::StateChange.create!(
+          target_state: 'passed',
+          target_uuid: tube_uuid,
+          user_uuid: user_uuid
+        )
       end
     end
 
     def tubes_from_transfer
-      @transfer
+      create_transfer!
```

Comment:
`@create_transfer` is the memoized backing store for the method.  When you implement getters, you should use them at all times if possible.  Not doing so can cause hard to debug issues where the getter is supposed to be managing access to the backing store.  The only reason the other mention in this file is not using the method is that you cannot check for a `nil` value if it is always populated by the getter method.

I don't think we really need to check for a `nil` value, but I didn't want to get too deep into the weeds and cause a bug myself.  The first time you try to get the transfer via the method, it would be called anyway, so why would you need to check if it's been called?  I'm not sure.